db/db_sst_test.cc

```diff
@@ -1629,6 +1631,44 @@ TEST_F(DBSSTTest, GetTotalSstFilesSize) {
   ROCKSDB_NAMESPACE::SyncPoint::GetInstance()->DisableProcessing();
 }
 
+TEST_F(DBSSTTest, OpenDBWithoutGetFileSizeInvocations) {
+  Options options = CurrentOptions();
+  options.disable_auto_compactions = true;
+  options.compression = kNoCompression;
+  options.enable_blob_files = true;
+  options.blob_file_size = 32;  // create one blob per file
+  options.skip_checking_sst_file_sizes_on_db_open = true;
+  options.env = MockEnv::Create(Env::Default());
```

Comment:
We could store the result of `MockEnv::Create` in a `unique_ptr` to eliminate the need for manually calling `delete`

Reply:
done